src/Hyperwallet/Hyperwallet.php

```diff
@@ -1803,4 +1805,57 @@ private function updateTransferMethod($userToken, $transferMethod, $transferMeth
                 ), $transferMethod, array());
     }
 
+    /**
+     * Create a bank card status transition
+     *
+     * @param string $userToken The user token
+     * @param UserStatusTransition $transition The status transition
+     * @return UserStatusTransition
+     *
+     * @throws HyperwalletArgumentException
+     * @throws HyperwalletApiException
+     */
+    public function updateVerificationStatus($userToken, $verificationStatus){
```

Comment:
Do we really need this parameter(verificationStatus) here? We can hardcode this value in the function instead of getting it as a parameter. Just a suggestion. Pls check.

Reply:
This is a generic function to handle any change in verificationstatus set later. More over your suggestion is more functional which is good, but we dont have all possible functional scenarios documented. I think this is the safest way to handle.